### ISSUE TITLE
problem_id format

### ISSUE THREAD

**Issue by GoogleCodeExporter** [numbbo/numbbo#87]
```
The current ID format for the implemented BBOB 2009 testbed is 

   bbob2009_fXX_iYY_dZZ

This seems not ambitious enough, as it does not allow for instance numbers or 
dimensions larger than 99. 

For the (noiseless) BBOB 2009 testbed the fXX id is sufficient.

Caveat: hard coded dependencies when the ID is changed (for this reason the 
change should not be postponed for too long).
```

Original issue reported on code.google.com by `nikolaus...@gmail.com` on 24 Feb 2015 at 9:07

**Comment by brockho** [comment]
As long as there are no dependencies, there is no big issue in leaving the current format. Otherwise, this issue has to be solved before solving #93

**Comment by oaelhara** [comment]
This specific format is currently used by suite_bbob2009_get_instance_id and suite_bbob2009_get_problem_id in suite_bbob2009.c.
These functions need to be updated to accommodate the less restrictive format.

**Comment by ttusar** [comment]
Closed by commit a3bc14288a5adfa5dbd3b4ea76a01ce5f9a3c41 to the development branch.
